src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/jpa/entity/PersonEntity.kt

```diff
@@ -27,75 +27,75 @@ class PersonEntity(
 
   @Id
   @GeneratedValue(strategy = GenerationType.IDENTITY)
-  val id: Long? = null,
+  var id: Long? = null,
 
   @Column
-  val title: String? = null,
+  var title: String? = null,
 
   @Column(name = "first_name")
-  val firstName: String? = null,
+  var firstName: String? = null,
 
   @Column(name = "last_name")
-  val lastName: String? = null,
+  var lastName: String? = null,
 
   @Column(name = "middle_names")
-  val middleNames: String? = null,
+  var middleNames: String? = null,
 
   @Column
-  @OneToMany(mappedBy = "person", cascade = [CascadeType.ALL], fetch = FetchType.LAZY)
+  @OneToMany(mappedBy = "person", cascade = [CascadeType.ALL], fetch = FetchType.EAGER, orphanRemoval = true)
```

Comment:
What's the thinking behind this change?

Reply:
OK - do we have a test which verifies this behaviour?

Reply:
Yes